app/controllers/media/objects_controller.rb

```diff
@@ -4,11 +4,17 @@ class Media::ObjectsController < ApplicationController
 
   def show
     style = params[:style]
-    send_file @media_object.item.path(style), type: @media_object.item_content_type, disposition: "inline"
+    @answer = @media_object.answer
```

Comment:
I'm realizing that I think we need to authorize access to these objects. So will need to add to ability, etc.